crates/utils/src/lib.rs

```diff
@@ -70,12 +70,175 @@ pub struct LemmyError {
   pub inner: anyhow::Error,
 }
 
-impl<T> From<T> for LemmyError
-where
-  T: Into<anyhow::Error>,
-{
-  fn from(t: T) -> Self {
-    LemmyError { inner: t.into() }
+impl From<deadpool_sync::InteractError> for LemmyError {
```

Comment:
This was very unfortunate: I had to get rid of the generic from impl for `LemmyError` because the error types for deadpool didn't transform easily to anyhow. This is more verbose but explicit anyway.